src/main/java/jenkins/plugins/gerrit/GerritApiBuilder.java

```diff
@@ -117,26 +117,23 @@ public GerritApiBuilder stepContext(StepContext context)
 
   public GerritApi build() {
     GerritApi gerritApi = null;
-    List<HttpClientBuilderExtension> extensions = new ArrayList<>();
-    GerritAuthData authData = null;
     if (gerritApiUrl == null) {
       logger.println("Gerrit Review is disabled no API URL");
-    } else if (username == null) {
-      logger.println("Gerrit Review is disabled no credentials");
-      authData = new AnonymousAuth(gerritApiUrl.toString());
```

Comment:
had you do not want to access gerrit anonymously, what is the reasoning of having AnonymousAuth?

Reply:
AnonymousAuth is for instantiating the Gerrit API without providing credentials.